components/Forms/FormAddPlaces.js

```diff
@@ -1,12 +1,14 @@
 import styled from "styled-components";
 import React from "react";
 import StyledTertiarySection from "../Sections/StyledTertiarySection";
+import UploadImage from "./FormImageUpload";
 
 import { useState } from "react";
 import useSWR from "swr";
 
 export default function AddPlaces({ getUniqueValues }) {
   const [isChecked, setIsChecked] = useState([]);
```

Comment:
the naming of this state indicates, that this is a Boolean value (Boolean state are always names "is..:"
You are maintaining an array of the activities that are checked to this would me more representative:
```suggestion
  const [checkedActivities, setCheckedActivities] = useState([]);
```

Reply:
I can't comment in the handleChange function but don't forget to adjust the names also in the handleChange function! :)